simulator/docs/sample/debuggable-scheduler/scheduler.yaml

```diff
@@ -1,11 +1,4 @@
 kind: KubeSchedulerConfiguration
 apiVersion: kubescheduler.config.k8s.io/v1
 clientConnection:
-  kubeconfig: kubeconfig.yaml
```

Comment:
why is it removed?

Reply:
I fix this diff.
[da41f80](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/353/commits/da41f800cee7789337bacfc3cfec1abd33713681)